test/helpers/data/app-workload-helper.js

```diff
@@ -102,3 +112,11 @@ module.exports.removeDependencies = function (inserts) {
     return knex(deletion.table).whereIn('id', deletion.id).del()
   })
 }
+
+var getMaxStagingId = function () {
```

Comment:
As discussed, this can be removed when we're using the `workload_report_id` as we'll have a sequenced unique identifier.